c/validation/adbc_validation.cc

```diff
@@ -53,6 +54,15 @@ namespace {
   } while (false)
 }  // namespace
 
+/// case insensitive string compare
+bool iequals(std::string s1, std::string s2) {
```

Comment:
move this into the anonymous namespace?

Reply:
also, try to use string_view here or at least const string&

Reply:
it's not using string_view or a const string because i'm actually modifying the strings here. I'll change this to do the comparison without modifying the strings and have it use string_view